tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/PrimitiveInjectionTest.java

```diff
@@ -282,4 +282,21 @@ public void verifyInjectedCustomString() throws Exception {
         Reporter.log(reply.toString());
         Assert.assertTrue(reply.getBoolean("pass"), reply.getString("msg"));
     }
+
+    @RunAsClient
+    @Test(groups = TEST_GROUP_CDI_PROVIDER,
+        description = "Verify that the injected customString claim is as expected")
+    public void verifyInjectedCustomBoolean() {
+        String uri = baseURL.toExternalForm() + "endp/verifyInjectedCustomBoolean";
```

Comment:
What is the difference with the added test in `ClaimValueInjectionTest.java`as it calls the same endpoint?

Reply:
The endpoint name is the same, yes, but the Resource class is different. One uses `PrimitiveInjectionEndpoint` (injects claims with primitives and wrapper types) and the other `ClaimValueInjectionEndpoint` (inject claims wrapper in `ClaimValue`). 

Since they are included selectively on each test class they are fine. If they were to be included both, then we will have a clash in the endpoints name. But this is the structure already in place. Maybe we should consider to change that?